doc/resources.md

```diff
@@ -44,6 +44,6 @@ See [this page](./github.md) for more information.
 
 - [Docker](https://hub.docker.com/u/nixos) - owner: infra
 - [Open collective](https://opencollective.com/nixos) - owner: foundation
-- [YouTube](https://www.youtube.com/@NixOS-Foundation) - owner: marketing
+- [YouTube](https://www.youtube.com/@NixOS-Foundation) - Google brand account; owners: @garbas (primary owner), @fricklerhandwerk (owner), @wamirez (manager)
```

Comment:
```suggestion
- [YouTube](https://www.youtube.com/@NixOS-Foundation) - Google brand account; owners: @garbas (primary owner), @fricklerhandwerk (owner), @djacu (owner), @wamirez (manager)
```

Still waiting to hear back if @garbas is handing off primary ownership but this is correct as of right now

Reply:
Still receive the github pings :)

I've set @djacu and @thilobillerbeck as Owner, but I'm not able to change primary ownership. I'm not sure how primary owner is set, I didn't see the option to do that. I think with owner role you should be able to set yourself as primary owner.